src/nnet3/nnet-simple-component.cc

```diff
@@ -2082,9 +2086,12 @@ void Convolutional1dComponent::InitFromConfig(ConfigLine *cfl) {
   int32 input_dim = -1, output_dim = -1;
   int32 patch_dim = -1, patch_step = -1, patch_stride = -1;
   cfl->GetValue("learning-rate", &learning_rate); //optional
+  cfl->GetValue("patch-dim", &patch_dim);
```

Comment:
@vijayaditya <https://github.com/vijayaditya> would you mind check the 
CI log? banned in China mainland....

On 10/19/2015 02:16 PM, Vijayaditya Peddinti wrote:
>
> In src/nnet3/nnet-simple-component.cc 
> <https://github.com/kaldi-asr/kaldi/pull/272#discussion_r42338739>:
>
> > @@ -2082,9 +2086,12 @@ void Convolutional1dComponent::InitFromConfig(ConfigLine *cfl) {
> >    int32 input_dim = -1, output_dim = -1;
> >    int32 patch_dim = -1, patch_step = -1, patch_stride = -1;
> >    cfl->GetValue("learning-rate", &learning_rate); //optional
> > +  cfl->GetValue("patch-dim", &patch_dim);
>
> Please modify to
>
> |ok = ok && cfl->GetValue("patch-dim", &patch_dim); ok = ok && 
> cfl->GetValue("patch-step", &patch_step); ok = ok && 
> cfl->GetValue("patch-stride", &patch_stride); |
>
> the method has to ensure that the specified values exist.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/272/files#r42338739>.
>